### PR TITLE
Registrant Parsing Improvement

### PR DESCRIPTION
**Pull request by danthony4**

### CODE REVIEW

**Comment by Steve-hyas** [comment]
@danthony4  How does this improve our data collection results? 
Can you give a before / after picture of what is returned to our whois collector and what new information will be stored and displayed in comox / hyas insight ( for domain retirementprogramtrinity-health.org )

**Comment by danthony4** [comment]
@Steve-hyas sure thing:

`...
Registrant Organization: leed johnny
Registrant State/Province: sichuansheng
Registrant Country: CN
...`

-> 

`{
  ...
  'contacts': {
    'registrant': {
      'organization': 'Leed Johnn', 
      'state': 'Sichuansheng', 
      'country': 'China'
    }, 
    'tech': None, 
    'admin': None, 
    'billing': None
  }
  ...
}`

File: pythonwhois/parse.py [hunk 1]
```diff
@@ -286,6 +286,7 @@ def allow_trailing_comma_dict(regexes):
 	"person:\s+(?P<name>.+)", # nic.ru (person)
 	"org:\s+(?P<organization>.+)", # nic.ru (organization)
   "Registrant:\n\s+Name:[ ]*(?P<name>.+)\n", # .ca
+  "Registrant Organization: (?P<organization>.+).+\n.*Registrant State/Province: (?P<state>.+)\n.*Registrant Country: (?P<country>.+)\n"
```
Comment (pvangool) [inline pythonwhois/parse.py]: Is the missing 'e' cause by "(?P<organization>.+)." instead of "(?P<organization>.+)"?

Comment (danthony4) [reply pythonwhois/parse.py]: Aha! Good catch. Pushing Fix!

**Pull request merged**